packages/flutter_tools/lib/src/build_system/targets/ios.dart

```diff
@@ -568,7 +598,11 @@ abstract class IosAssetBundle extends Target {
   const IosAssetBundle();
 
   @override
-  List<Target> get dependencies => const <Target>[KernelSnapshot(), InstallCodeAssets()];
+  List<Target> get dependencies => const <Target>[
+    CheckDevDependenciesIos(),
```

Comment:
Nit: I know `DebugIosApplicationBundle` uses super to also get these dependencies, but I'd add it directly to `DebugIosApplicationBundle` dependencies just to keep it consistent with Profile and Release

Reply:
That's definitely easier to read, good idea. Updated!